components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultRequestCoordinator.java

```diff
@@ -965,6 +965,8 @@ protected void findPreviousAuthenticatedSession(HttpServletRequest request, Auth
         }
 
         context.setServiceProviderName(effectiveSequence.getApplicationConfig().getApplicationName());
+        context.setServiceProviderResourceId(effectiveSequence.getApplicationConfig().
```

Comment:
Hope none of these configs return null

Reply:
Ideally it shouldn't but added a null check to avoid the risk.
https://github.com/wso2/carbon-identity-framework/pull/5548/commits/85ad752df79d5f0269fbb12b14605bb0a4489639